client/src/components/MeetingExperience.tsx

```diff
@@ -75,12 +89,26 @@ export const MeetingExperience = (props: MeetingExperienceProps): JSX.Element =>
 
     _createAdapters();
   }, [credential, displayName, endpointUrl, locator, userId, onDisplayError]);
-
   if (callWithChatAdapter) {
     const logo = logoUrl ? <img style={meetingExperienceLogoStyles} src={logoUrl} /> : <></>;
     const locale = COMPOSITE_LOCALE_EN_US;
     const formFactorValue = new MobileDetect(window.navigator.userAgent).mobile() ? 'mobile' : 'desktop';
 
+    if (renderPostCall) {
+      return (
+        <Survey
+          postCall={postCall}
+          onRejoinCall={() => {
+            callWithChatAdapter.joinCall();
+            callWithChatAdapter.onStateChange((state: CallWithChatAdapterState) => {
+              if (state.page === 'configuration') state.page = 'lobby';
```

Comment:
We should probably add a comment about why this is necessary.

Reply:
have made this change.